.eslintrc

```diff
@@ -1,15 +1,17 @@
 {
   "parser": "babel-eslint",
-  "extends": "airbnb/base",
+  "extends": "airbnb-base",
   "env": {
     "mocha": true
   },
   "globals": {
-    "expect": false
+    "expect": false,
+    "window": true
```

Comment:
what does `window` do?

Reply:
Adds `window` to the list of accepted globals across all files so that eslint doesn't warn that "window is undefined" or whatever.